src/web/mjs/connectors/templates/SpeedBinb.mjs

```diff
@@ -160,22 +164,26 @@ export default class SpeedBinb extends Connector {
     _getPageList_v016130( chapterID, apiURL ) {
```

Comment:
either make a new method which implements the hobbyjapan behavior, or overwrite this method in hobbyjapan connector itself